doc/source/commands/keg.rst

```diff
@@ -1,28 +1,73 @@
 keg
 ===
 
+.. _keg_synopsis:
+
 SYNOPSIS
 --------
 
 .. code:: bash
 
+   keg (-r RECIPES_ROOT|--recipes-root=RECIPES_ROOT)
+       [-a ADD_DATA_ROOT] ... [-d DEST_DIR] [-fv]
+       SOURCE
+   keg -h | --help
    keg -h | --help
 
 DESCRIPTION
 -----------
 
-Description
+Keg is a tool which helps to create and manage image descriptions suitable
+for the `KIWI <https://osinside.github.io/kiwi/>`__ appliance builder. Its
+main use case is to keep control over a larger amount of image descriptions
+and prevent duplication of description data.
```

Comment:
s/Its main use...description data/While keg can be used to manage a single image definition the tool provides no considerable advantage in such a use case. The primary use case for keg are situations where many image descriptions must be managed and the image descriptions have considerable over lap with respect to content and setup./

Reply:
done